content/en/blog/_posts/2022-04-28-cni-version-related-breaking-change.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: blog
+title: "Breaking changes related to CNI version in popular container runtimes"
+date: 2022-04-28T12:00:00+0000
+slug: cni-version-related-breaking-change-k8s-1.24
+canonicalUrl: https://kubernetes.dev/blog/2022/04/28/cni-version-related-breaking-change-k8s-1.24/
+---
+
+**Authors & Contributors:** [Mike Brown](https://github.com/mikebrow), [Rolfe Dlugy-Hegwer](https://github.com/rolfedh),[Nate W.](https://github.com/nate-double-u), and others.
+
+---
+
+## Introduction
+
+TL;DR: Before you upgrade to Kubernetes 1.24, if you use a CNI network plugin that complies with Container Network Interface (CNI) versions earlier than v1.0.0 and you plan to use any of the following container runtimes, you must specify the CNI version. Otherwise, your network will break.
```

Comment:
```suggestion
_TL;DR: Before you upgrade to Kubernetes 1.24, if you use a CNI network plugin that complies
with Container Network Interface (CNI) versions earlier than v1.0.0 and you plan to use either
of the following container runtimes, you must specify the CNI version. Otherwise, your cluster
network will break._
```

Maybe change “complies with” to “is set up to use”?

Reply:
Updated. Thanks.